google/ggrc-core
# QUICK-FIX Squash migrations

# Issue description

Since moving all migrations to a single module, we don't support downgrade to 1.19.0 or lower. Now the old migrations just take unneeded developer time.

# Steps to test the changes

This change should only affect database on a clean migration so from an empty database. we must make sure that the database state is identical if we reset the database and after this migration.

# Solution description

Create a new dump and set it as the seed for the initial migration.
The migration utils have not been touch even though some of them are now obsolete. Since more than a few will have to remain for future migrations each util cleanup will be done separately to ensure we don't delete anything that we might need.

# Sanity checklist



# Migration checklist

# PR Review checklist
